content/src/fieldConfig/config.json

```diff
@@ -68,7 +68,11 @@
     "navBarUnnamedBusinessText": "Unnamed",
     "taskPageNavBarHeading": "Business Roadmap",
     "needsAccountSnackbarTitleExistingAccount": "Link Your myNewJersey Account to Save Your Progress",
-    "needsAccountModalBody": "To use this feature, you need to complete your registration with the State of New Jersey's secure login system, MyNewJersey. This will ensure your privacy, and allow you to come back any time."
+    "needsAccountModalBody": "To use this feature, you need to complete your registration with the State of New Jersey's secure login system, MyNewJersey. This will ensure your privacy, and allow you to come back any time.",
+    "navBarMyAccountText": "My Account",
```

Comment:
dust - we talked about not adding to this file, instead we discussed carving out the areas of this confid into their own collection / with another collection.

Reply:
I can do that in a follow up ticket!